include/ensmallen_bits/callbacks/progress_bar.hpp

```diff
@@ -216,10 +216,10 @@ class ProgressBar
         output << ".";
       }
     }
-
     const size_t stepTime = epochTimer.toc() / (double) epochSize * 1000;
-    output << "] " << progress << "% - " << (size_t) epochTimer.toc() % 60
-        << "s " << stepTime << "ms/step " << "- loss: " << objective  <<  "\n";
+    output << "] " << progress << "% - " << epochTimer.toc()
```

Comment:
There's a subtle bug here.  `epochTimer.toc()` is called twice: here and on the preceding line.  Note that the timer is still running between the two calls to `.toc()`, so it's entirely possible that the second call to `epochTimer.toc()` will provide a different value than the first call to `epochTimer.toc()`.  A lot of stuff may get executed between the two calls, especially when streams are involved (ie. the `output << ...` part).

To fix this, need to store the value of `epochTimer.toc()` into a separate variable and use that in further calculations.  For example:
```
  const double epochTimer_elapsed = epochTimer.toc();
  const size_t stepTime = epochTimer_elapsed / (double) epochSize * 1000;
  output << "] " << progress << "% - " << epochTimer_elapsed
```

Ref: docs for [wall_clock](http://arma.sourceforge.net/docs.html#wall_clock)